multi-node.sh

```diff
@@ -11,6 +11,7 @@ libfabric_job_type=${libfabric_job_type:-"master"}
 # Current LibfabricCI IAM permissions do not allow placement group creation,
 # enable this after it is fixed.
 # export ENABLE_PLACEMENT_GROUP=1
+export USER_DATA_FILE=${USER_DATA_FILE:-${JENKINS_HOME}/user_data_script.sh}
```

Comment:
Both Shi and I had a concern about variable ${JENKINS_HOME}, just to make sure that you two have sync up, and already had an agreement here.

Reply:
I have talked with @wckzhang offline about this and this change looks good to me.

Reply:
Yes, so there are a couple checks now. If USER_DATA_FILE exists, use that, if it doesn't use JENKINS_HOME/user_data_script.sh If this file doesn't exist, the run-instances command won't use this argument and there's no harm.